Connection/RESTConnector.cs

```diff
@@ -488,6 +488,10 @@ private IEnumerator ProcessRequestQueue()
                             Log.Error("RESTConnector.ProcessRequestQueue()", "Exception when initializing WWWForm: {0}", e.ToString());
                         }
                         unityWebRequest = UnityWebRequest.Post(url, form);
+                        if (req.HttpMethod == UnityWebRequest.kHttpVerbPUT) 
+                        {
+                            unityWebRequest.method = "PUT";
+                        }
```

Comment:
This seems a bit inconsistent btw post and put

does this work?

```cs
if (req.HttpMethod == UnityWebRequest.kHttpVerbPOST)
{
    unityWebRequest = UnityWebRequest.Post(url, form);
}
if (req.HttpMethod == UnityWebRequest.kHttpVerbPUT)
{
    unityWebRequest = UnityWebRequest.Put(url, form);
}

```

Reply:
@mediumTaj Put doesn't allow formdata in the second parameter